### PR TITLE
Updated Base ranged range and MS

### PR DESCRIPTION
**Pull request by adocoD**
Hunter melee and ranged only had a difference of 3 yds. It should actually be 7.

Hunter range to boss gets calculated from edge of hitbox. Since melee range in the sim is 5 (4+1), ranged would be 12 (4+8).

Movement speed was also incorrect , so changed it to 7yd per second.

### CODE REVIEW

File: sim/core/unit.go [hunk 1]
```diff
@@ -428,7 +428,7 @@ func (unit *Unit) MoveTo(moveRange float64, sim *Simulation) {
 	unit.moveSpell.Cast(sim, unit.CurrentTarget)
 
 	sim.AddPendingAction(NewPeriodicAction(sim, PeriodicActionOptions{
-		Period:          time.Millisecond * 500,
+		Period:          time.Millisecond * 143,
```
Comment (kayla-glick) [inline sim/core/unit.go:431]: Why 143ms? That's an oddly specific number 😄

File: sim/hunter/aimed_shot.go [hunk 2]
```diff
@@ -56,7 +56,7 @@ func (hunter *Hunter) getAimedShotConfig(rank int, timer *core.Timer) core.Spell
 			},
 		},
 		ExtraCastCondition: func(sim *core.Simulation, target *core.Unit) bool {
-			return hunter.DistanceFromTarget >= 8
+			return hunter.DistanceFromTarget >= 12
```
Comment (kayla-glick) [inline sim/hunter/aimed_shot.go:59]: Maybe we could pull this out into a constant value in `sim/core/constants`? `core.MinRangedAttackDistance`

File: sim/core/unit.go (hunk 1 above)
Comment (adocoD) [reply sim/core/unit.go:431]: Time per yd moved. I suppose I could've written 1000 / 7 there to be accurate, since it's 7 yd/s.

Comment (adocoD) [reply sim/core/unit.go]: Fixed this

File: sim/hunter/aimed_shot.go (hunk 2 above)
Comment (adocoD) [reply sim/hunter/aimed_shot.go]: Added constant for both MaxMeleeAttackDistance and MinRangedAttackDistance

**Pull request merged**